src/commands/schemas.ts

```diff
@@ -270,12 +270,9 @@ async function deleteSchemaVersionCommand(schema: Schema) {
       // already deleted in other workspace or other means?
       if (e.response.status !== 404) {
         // not a 404, something unexpected/
-        logError(
-          e,
-          "Error fetching schemas for subject while deleting schema version",
-          { subject: schema.subject },
-          true,
-        );
+        logError(e, "Error fetching schemas for subject while deleting schema version", {
+          extra: { subject: schema.subject },
```

Comment:
Need some kind of "send to sentry" sentinel value here like `{ error: {} }` that still passes the "sentryContext has keys" check in `logError` ⬆️